cmake/ctest/drivers/atdm/ride/local-driver.sh

```diff
@@ -8,9 +8,13 @@ if [ "${Trilinos_CTEST_DO_ALL_AT_ONCE}" == "" ] ; then
   export Trilinos_CTEST_DO_ALL_AT_ONCE=TRUE
 fi
 
+if [ "${ATDM_CONFIG_QUEUE}" == "" ] ; then
+  export ATDM_CONFIG_QUEUE=rhel7F
```

Comment:
Okay, I think I see the problem.  The problem might be that we only source the script `load-env.sh` and therefore `ride/enviornment.sh` on the compute node after we need to know the queue.

Let's look at refactoring the Jenkins drivers to source the full script `load-env.sh` up front even before running the local-driver.sh script.  We could source this is the `smark-jenkins-driver.sh` script.  I think every system that we are building on should allow sourcing `atdm/ load-env.sh` on the login node, even if it will be sourced again on the compute node.

This would eliminate a lot of duplication here.